klippy/klippy.py

```diff
@@ -57,8 +57,19 @@ def __init__(self, input_fd, bglogger, start_args):
         self.in_shutdown_state = False
         self.run_result = None
         self.event_handlers = {}
-        gc = gcode.GCodeParser(self, input_fd)
-        self.objects = collections.OrderedDict({'gcode': gc})
+        web_hooks = webhooks.WebHooks(self)
+        gc = gcode.GCodeParser(self, input_fd, web_hooks)
+        self.objects = collections.OrderedDict(
+            {'gcode': gc, 'webhooks': web_hooks})
```

Comment:
FYI, creating the OrderedDict from a dictionary with two items makes the ordering between those two items unpredictable.  I'd recommend something like this instead:
```python
        self.objects = collections.OrderedDict()
        self.objects['webhooks'] = webhooks = webhooks.WebHooks(self)
        self.objects['gcode'] = gc = gcode.GCodeParser(self, input_fd)
```

Also, minor, but then gcode.py can lookup the webhook object from the main printer.lookup_object() call.